packages/web-react/src/components/ModalComposed/ModalComposedDialog.tsx

```diff
@@ -1,18 +1,44 @@
-import React, { ElementType, ForwardedRef, forwardRef } from 'react';
+import React, { CSSProperties, ElementType, ForwardedRef, forwardRef } from 'react';
 import classNames from 'classnames';
 import { useStyleProps } from '../../hooks';
 import { ModalComposedDialogProps, ModalComposedDialogElementType } from '../../types/modalComposed';
 import { useModalComposedStyleProps } from './useModalComposedStyleProps';
 
+interface CSSPropertiesWithVariables extends CSSProperties {
```

Comment:
In this case, I am also suggesting a more descriptive name, `CustomizedHeightCSSProperties`... `CSSPropertiesWithVariables` is too general for this case, IMHO. We are dealing with height customization, so I would use that word in the naming. :-)